ef4/decorator-transforms
# Re-add browser support via removing tsup

- Reverts: https://github.com/ef4/decorator-transforms/pull/44
  - Which Reverts: https://github.com/ef4/decorator-transforms/pull/37
    - Which caused: https://github.com/ef4/decorator-transforms/issues/42

Related discussion:
- https://github.com/ef4/decorator-transforms/pull/43
- https://github.com/ef4/decorator-transforms/pull/27


Manual Testing (all successes):
- `pnpm test` 
- run `pnpm dev:browser` (or `pnpm vite`) -- see output as in #37 
- pnpm link from a v2 addon to use this built code in the v2 addon, and that v2 addon should build successfully (verified locally)
